core/Cargo.toml

```diff
@@ -19,7 +19,7 @@ instant = "0.1.11"
 libp2p-identity = { version = "0.1", path = "../identity", features = ["peerid", "ed25519"] }
 log = "0.4"
 multiaddr = { version = "0.17.0" }
-multihash = { version = "0.17.0", default-features = false, features = ["std", "multihash-impl", "identity", "sha2"] }
+multihash = { version = "0.17.0", default-features = false, features = ["std"] }
```

Comment:
We re-export `multihash` so technically, this is a breaking change. I am tempted to release it as non-breaking as I'd argue that users should enable features for re-exported dependencies themselves.